LottieUWP/LottieDrawable.cs

```diff
@@ -68,6 +68,43 @@ public LottieDrawable()
             Unloaded += UserControl_Unloaded;
         }
 
+        public CanvasCommandList GetCanvasImage(ICanvasResourceCreator resourceCreator, float scaleX, float scaleY)
+        {
+            lock (this)
+            {
+                var commandList = new CanvasCommandList(resourceCreator);
+                using (var session = commandList.CreateDrawingSession())
+                {
+                    var width = _composition.Bounds.Width * scaleX;
+                    var height = _composition.Bounds.Height * scaleY;
+                    if (_bitmapCanvas == null || _bitmapCanvas.Width < width || _bitmapCanvas.Height < height)
```

Comment:
Why only when less than **<**, not different **!=**?

Reply:
Because I'm using this condition to make sure that the image doesn't get blurry when I zoom in